src/FatLib/FatVolume.cpp

```diff
@@ -602,7 +603,7 @@ bool FatVolume::wipe(print_t* pr) {
     goto fail;
   }
   if (!writeBlock(m_fatStartBlock, cache->data) ||
-      (fbs->fatCount == 2 && !writeBlock(m_fatStartBlock + m_blocksPerFat, cache->data))) {
+      (m_fatCount == 2 && !writeBlock(m_fatStartBlock + m_blocksPerFat, cache->data))) {
```

Comment:
`fbs->fatCount` is local variable of init(), we need to store this (like other information of boot sector) to class variable.

Reply:
as far as I can see, fbs isn't even defined, it needs to be: cache->fbs->fatCount
but you are right, this needs to be a class variable.